### DISCUSSION TITLE
how to save the segmentation model?

### DISCUSSION THREAD

**Discussion by TSbme** [Project-MONAI/tutorials#1202] [Q&A]
After training, I can't find the best_model.pth. I think the code just calculates the results without saving the model. How to save it?

**Comment by KumoLiu** [comment]
Hi @tangqishi, you can refer to this [tutorial](https://github.com/Project-MONAI/tutorials/blob/main/3d_segmentation/brats_segmentation_3d.ipynb) and find how to save model.
```
if metric > best_metric:
      best_metric = metric
      torch.save(
          model.state_dict(),
          os.path.join(root_dir, "best_metric_model.pth"),
      )
      print("saved new best metric model")
```
Hope it can help you, thanks!

**Reply by TSbme** [reply]
Thanks so much！
